contingency/contingency-api/src/main/java/com/powsybl/contingency/ContingencyContextType.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.contingency;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ *
+ * Define for a contingencyContext the type of information asked.
+ * It can be n situation (N_SITUATION), n-k situation on a specific contingency (SPECIFIC) or both (ALL)
+ */
+
+public enum ContingencyContextType {
+    ALL,
```

Comment:
Better place the javadoc on each value, like
```java
/**
 *  Corresponds to all contingencies and pre-contingency situation
 */
ALL,
...
```
This way we get the correct doc when we ask the doc for one value (in your IDE for example).